roles/lib_openshift/src/class/oc_env.py

```diff
@@ -0,0 +1,147 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCEnv(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+
+    container_path = {"pod": "spec.containers[0].env",
+                      "dc":  "spec.template.spec.containers[0].env",
+                      "rc":  "spec.template.spec.containers[0].env",
+                     }
+
+    # pylint allows 5. we need 6
+    # pylint: disable=too-many-arguments
```

Comment:
@abutcher @detiber, Would also like feedback on number of params.  Its hard when mapping calls with 5+ parameters when self is used as one.  Then you get into an abstraction layer where you pass around a dto/dictionary where the data ultimately lives.  @ashcrow, thoughts on this?